packages/cli/src/lib/collect/collect-command.ts

```diff
@@ -18,6 +18,12 @@ export function yargsCollectCommandObject(): CommandModule {
       console.info(chalk.bold(CLI_NAME));
       console.info(chalk.gray(`Run ${command}...`));
       await collectAndPersistReports(options);
+      if (options.categories.length === 0) {
+        console.info(
+          chalk.gray(`💡Configure categories to see the scores in an overview table.
+        See: https://github.com/code-pushup/cli/blob/main/packages/cli/README.md`),
```

Comment:
Is the indentation here intentional? Whitespace in template strings is printed "as is" (including line breaks), so this would print:

```
💡Configure categories to see the scores in an overview table.
        See: https://github.com/code-pushup/cli/blob/main/packages/cli/README.md
```

This would print it on one line (if you want only the source code to span multiple lines, use `+` concatenation):

```suggestion
          chalk.gray('💡 Configure categories to see the scores in an overview table. See: https://github.com/code-pushup/cli/blob/main/packages/cli/README.md'),
```

Reply:
Made it into a one-liner. Added it to both autorun and collect command.